mapsims/noise.py

```diff
@@ -67,6 +67,19 @@ def _band_index(tube, band):
     return so_utils.tubes[tube].index(band)
 
 
+def _get_wscale_factor(white_noise_rms, tube, sky_fraction):
+    """Internal function to re-scale white noise power
+    to a new value corresponding to white noise RMS in uK-arcmin.
+    """
+    if white_noise_rms is None:
+        return np.ones((2, 1))
+    cnoise = np.sqrt(
+        self.get_white_noise_power(tube, sky_fraction=1, units="arcmin2") * sky_fraction
```

Comment:
it looks like here it is referencing `self`

Reply:
I just meant that the get_white_noise_power function needs to be a member function of the class because it access members of the class when it runs.

Reply:
but the code is currently broken, can you revert?

Reply:
Ah missed that. So looks like neither function can sit outside the class.

Reply:
Fixed.